### ISSUE TITLE
05/10/23_NYP_Android_Go To_Driver is getting metro cancel pop-up image when he got GOTO+Accessibility ride _Parent Id - 1980

### ISSUE THREAD

**Issue by AlluGowthami28** [nammayatri/nammayatri#3494]
#1980 
[Driver is getting metro cancel pop-up image when he got GOTO+Accessibility ride](https://drive.google.com/file/d/1A52aujuWXLEOvCIFgT8yyFQWu4qMM6hI/view?usp=sharing)

**Steps to reproduce**
1. Accept a ride with Goto+Accessibility tag
2. Cancel the ride and observe the image

**Actual result**
Driver is getting metro cancel pop-up image when he got GOTO+Accessibility ride

**Expected result**
Accessibility cancel pop up should be there.

**Comment by Rohit4997** [comment]
@AlluGowthami28  There is not Accessibility cancel pop up, So made it default

**Comment by krupal-14** [comment]
@Prithivirajkota Can you please  confirm this once.

**Comment by Prithivirajkota** [comment]
was source or destination metro? for cancellation priority, can we consider Go to >> metro >> normal cancellation? @Rohit4997 @krupal-14

**Comment by krupal-14** [comment]
Yes @Prithivirajkota source was metro location.

**Comment by krupal-14** [comment]
Retested,Working fine.
